cpp/ql/src/semmle/code/cpp/controlflow/SubBasicBlocks.qll

```diff
@@ -81,6 +81,15 @@ class SubBasicBlock extends ControlFlowNodeBase {
     )
   }
 
+  /**
+   * DEPRECATED: use `getRankInBasicBlock` instead. Note that this predicate
+   * returns a 0-based position, while `getRankInBasicBlock` returns a 1-based
+   * position.
+   */
+  deprecated int getPosInBasicBlock(BasicBlock bb) {
```

Comment:
Why are we adding a deprecated predicate?  Is there another part to this change?

Reply:
... ah, looks like I was viewing an old version of the PR, or possibly a single commit.